components/TodaysCard/index.js

```diff
@@ -18,16 +23,23 @@ const StyledList = styled.li`
     margin: 2px;
     margin-left: 8px;
   }
+  &:hover {
+    background-color: lightslategray;
+  }
 `;
 
 export default function TodaysCard({ event }) {
   return (
     <>
       <StyledList>
-        <h2>{event.title}</h2>
-        <p>{event.time}</p>
-        <p>{event.country}</p>
-        <p>{event.impact}</p>
+        <StyledLink href={`/events/${event.id}`}>
```

Comment:
Instead of making the whole thing clickable, just make the title so ^^

This way one can select text inside the link for copy-pasting elsewhere for instance.

Also the spec in HTML5 is not super clear about allowing `<a>`s to contain block elements - I am conservative and I am against that!